src/components/Cartjs/MetricChart.js

```diff
@@ -0,0 +1,179 @@
+import { useEffect, useMemo, useRef, useState } from 'react'
+import { Chart } from 'chart.js/auto'
+import classnames from 'classnames'
+
+import Loader from '../../common/Loader/Loader'
+
+import { formatNumber } from '../DetailsMetrics/detailsMetrics.utils'
+import {
+  calculateMaxTicksLimit,
+  customTooltip,
+  formatYaxisForBarChart,
+  hexToRGB
+} from './metricChart.util'
+
+const GenericMetricChart = ({ chartConfig, showGrid }) => {
+  const chartRef = useRef(null)
+  const chartInstance = useRef(null)
+  const [isLoading, setIsLoading] = useState(true)
+  const canvasClassNames = classnames(isLoading && 'hidden')
```

Comment:
It's not recommended to use generic classes as `hidden`. Better to have it as modificator: <main-class>_hidden

Reply:
change classname